helpdesk-service/helpdesk-service-server/src/test/resources/jbehave.stories/membershipEndpoints.story

```diff
@@ -0,0 +1,52 @@
+Meta:
+
+Narrative:
+As a user
+I want to save and recover membership data
+So that I can assert those actions are available and working.
+
+Scenario: Create an agent membership without id
+When I call the create membership API for agent without id
+Then receive a bad request error
+
+Scenario: Create an agent membership without gourp id
+When I call the create membership API for agent without group id
+Then receive a bad request error
```

Comment:
Please include a new step to verify the HTTP response and make sure the message is correct.